api/types/sandbox.proto

```diff
@@ -41,11 +41,14 @@ message Sandbox {
 	// bundle directory (similary to OCI spec).
 	google.protobuf.Any spec = 3;
 	// Labels provides an area to include arbitrary data on containers.
-	map<string, string> labels  = 4;
+	map<string, string> labels = 4;
 	// CreatedAt is the time the container was first created.
 	google.protobuf.Timestamp created_at = 5;
 	// UpdatedAt is the last time the container was mutated.
 	google.protobuf.Timestamp updated_at = 6;
 	// Extensions allow clients to provide optional blobs that can be handled by runtime.
 	map<string, google.protobuf.Any> extensions = 7;
+	// Sandboxer is the name of the sandbox controller who manages the sandbox.
+	string sandboxer = 10;
```

Comment:
It's ok to have 10 here. Since we don't guarantee the API in v1.7, we can revisit all the fields and reorder it.
It's like snapshotter. I prefer to put the sandboxer(or controllerName) as first-class field here. 

I am not a fan of using empty value as builtin things because it's too vague.

Reply:
Yes, I think the only place that we should set a default value to sandboxer is the config.toml default setting. and it is set to ModePodSandbox. I will check if the legacy pods works well after containerd is upgraded.